Dockerfile

```
@@ -0,0 +1,23 @@
+FROM ghcr.io/kbase/cdm-spark-standalone:pr-25
```

```suggestion
FROM --platform=linux/amd64 ghcr.io/kbase/cdm-spark-standalone:pr-25
```

We only publish an AMD64 build (no Mac/ARM version), it would not work for Mac by default.